In [1]:
import pandas as pd
import numpy as np

# ÖDEV1
Kredi limiti (creditlimit) 100.000'den büyük olan tüm müşterilerin (-customers tablosundan) müşteri numarası, adı ve kredi limit bilgilerini creditlimit alanına göre sıralı bir şekilde gösteriniz.

In [3]:
cevap1= pd.read_csv("sqlödev1.csv")
cevap1

,customerNumber,customerName,creditLimit
0,157,Diecast Classics Inc.,100600.0
1,319,Mini Classics,102700.0
2,148,"Dragon Souveniers, Ltd.",103800.0
3,458,"Corrida Auto Replicas, Ltd",104600.0
4,239,Collectable Mini Designs Co.,105000.0
5,321,Corporate Gift Ideas Co.,105000.0
6,276,"Anna's Decorations, Ltd",107800.0
7,496,Kelly's Gift Shop,110000.0
8,249,Amica Models & Co.,113000.0
9,363,Online Diecast Creations Co.,114200.0


# ÖDEV2
customers tablosundan customernumber bilgisi 151 olan kaydın her sütununu gösteriniz.

In [4]:
cevap2= pd.read_csv("sqlödev2.csv")
cevap2

,customerNumber,customerName,contactLastName,contactFirstName,phone,addressLine1,addressLine2,city,state,postalCode,country,salesRepEmployeeNumber,creditLimit
0,151,Muscle Machine Inc,Young,Jeff,2125557413,4092 Furth Circle,Suite 400,NYC,NY,10022,USA,1286,138500.0


# ÖDEV3 
Posta kodu (postalcode) 10022 olan müşterilerin kredi limit bilgilerini azalan sırada gösteriniz.

In [5]:
cevap3= pd.read_csv("sqlödev3.csv")
cevap3

,customerName,creditLimit,postalCode
0,Muscle Machine Inc,138500.0,10022
1,Land of Toys Inc.,114900.0,10022
2,Vitachrome Inc.,76400.0,10022
3,Classic Legends Inc.,67500.0,10022
4,Microscale Inc.,39800.0,10022
